### ISSUE TITLE
loadouts not saving in ace arsenal

### ISSUE THREAD

**Issue by SirClips** [acemod/ACE3#9562]
Arma 3: 2.14.150957 stable
CBA: 3.16.0.231017 stable
ACE3: 3.16.0 stable

for ace arsenal:

anytime I try to save a loadout, it never saves

basically I "Save loadout" and when I switch tabs its gone like I never saved it

**Comment by PabstMirror** [comment]
Will be fixed by https://github.com/acemod/ACE3/pull/9523
in the meantime you can go into editor and then tools->debug console and exec
```
profileNamespace setVariable ["ace_arsenal_saved_loadouts", []];
saveProfileNamespace
```

**Comment by jonpas** [comment]
Fixed by #9523.
